samples/bot-feed-members/nodejs/README.md

```diff
@@ -0,0 +1,144 @@
+---
+page_type: sample
+description: This sample showcases how apps function within federated group chats.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "08-21-2024 20:15:25"
+urlFragment: officedev-microsoft-teams-samples-bot-feed-members-nodejs
+---
+
+# Apps in federated
+
+Apps in federated group chats.
```

Comment:
Add proper description
https://review.learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/shared-channels?branch=pr-en-us-10792&tabs=bot

Reply:
Fixed